support/curses-helpers.c

```diff
@@ -154,3 +154,18 @@ int keyBackspace() {
   return KEY_BACKSPACE;
 }
 
+int print(const char* fmt, const char* arg) {
+  return printw(fmt, arg);
+}
+
+int printWindow(WINDOW* win, const char* fmt, const char* arg) {
+  return wprintw(win, fmt, arg);
+}
+
+int mvPrint(int x, int y, const char* fmt, char* arg) {
+  return mvprintw(x, y, fmt, arg);
+}
+
+int mvPrintWindow(WINDOW* win, int x, int y, const char* fmt, char* arg) {
+  return mvwprintw(win, x, y, fmt, arg); 
+}
```

Comment:
These functions take `y` before `x` (or `row` before `column`). That is the case for the ncurses functions themselves and I think it makes sense to repeat the convention in the FFI (i.e. not to flip it within the C helpers & Idris FFI definitions).

Of course, this will have no functional impact since it is just swapping the two names around, but it'll be less confusing to the next person who reads this code 😄.

```suggestion
int mvPrint(int y, int x, const char* fmt, char* arg) {
  return mvprintw(y, x, fmt, arg);
}

int mvPrintWindow(WINDOW* win, int y, int x, const char* fmt, char* arg) {
  return mvwprintw(win, y, x, fmt, arg); 
}
```